src/components/hooks/graphql/useTasksQuery.ts

```diff
@@ -0,0 +1,74 @@
+import { gql, useQuery } from '@apollo/client';
+
+export const GET_ALL_TASKS = gql`
+  query allTasks {
+    allTasks(condition: { isDeleted: false }) {
+      edges {
+        node {
+          taskId
+          title
+          dueDate
+          madeBy
+          owner
+          statusId
+          subTasksByParentTaskId(condition: { isDeleted: false }) {
+            nodes {
+              parentTaskId
```

Comment:
why do you need to get the parentTaskId ? you already select all the children of the task so the parentId seems not relevant